### PR TITLE
Added migration tools

### PR DESCRIPTION
**Pull request by Theresa-Nguyen**

### CODE REVIEW

File: ms-docs/JBoss EAP on Azure.MD [hunk 1]
```diff
@@ -1,109 +1,133 @@
...
+* JBoss EAP 7.2 on RHEL 8.0 VM (BYOS)
+* JBoss EAP 7.3 on RHEL 8.0 VM (BYOS)
+
+Along with Azure Marketplace offers, there are Quickstart templates made available for you to get started on your Azure migration journey. These Quickstarts include pre-built Azure Resource Manager(ARM) templates and script to deploy JBoss EAP on RHEL in various configurations and version combinations. You'll have: 
+
+* Load Balancer (LB) 
+* Private IP for load balancing and VMs
+* Virtual Network (VNET) with a single subnet
+* VM configuration (cluster or stand-alone)
+* Network Security Group (NSG)
```
Comment (Suraj2093) [inline ms-docs/JBoss EAP on Azure.MD:66]: We do not have NSG in the QS templates, so I have removed this point.

File: ms-docs/JBoss EAP on Azure.MD [hunk 2]
```diff
@@ -133,35 +157,37 @@ You can deploy the template in three following ways:
...
+* <a href="https://azure.microsoft.com/en-us/resources/templates/jboss-eap-clustered-vmss-rhel/"> JBoss EAP on RHEL (clustered, Virtual Machine Scale Set)</a> - This Azure template deploys a web application called eap-session-replication on JBoss EAP 7.2 or 7.3 cluster running on RHEL 7.7 or 8.0 Virtual Machine Scale Set instances.
 
 ## Resource Links:
 
-* Explore [JBoss EAP on Azure Red Hat OpenShift](https://azure.microsoft.com/services/openshift/)
-* Explore [JBoss EAP on App Service Linux](https://docs.microsoft.com/azure/app-service/quickstart-java)
-* [Azure for Java developers documentation](https://github.com/JasonFreeberg/jboss-on-app-service)
+* [Azure Hybrid Benefits)(https://docs.microsoft.com/azure/virtual-machines/windows/hybrid-use-benefit-licensing)
+* [Configuring a Java app for Azure App Service](https://docs.microsoft.com/azure/app-service/configure-language-java?pivots=platform-linux)
```
Comment (Suraj2093) [inline ms-docs/JBoss EAP on Azure.MD:181]: Changed this ink to "https://docs.microsoft.com/azure/app-service/configure-language-java"

**Comment by Suraj2093** [comment]
I have made these changes in the MS Doc and added a couple of comments. Please review.

File: ms-docs/JBoss EAP on Azure.MD [hunk 3]
```diff
@@ -1,109 +1,133 @@
...
 
-### Migrate from on-premises to Azure
+#### Red Hat Migration Toolkit for Applications (MTA)
+MTA is a migration tool for Java application servers. Use this tool to migrate from another app server to JBoss EAP. It works with IDE plugins for [Eclipse IDE](https://www.eclipse.org/ide/), [Red Hat CodeReady Workspaces](https://developers.redhat.com/products/codeready-workspaces/overview), and Visual Studio Code ([VS Code])(https://code.visualstudio.com/docs/languages/java) for Java.  MTA is a free and open-source tool that will:
```
Comment (Suraj2093) [inline ms-docs/JBoss EAP on Azure.MD:41]: Edited the hyperlink mapping for VS Code.

File: ms-docs/JBoss EAP on Azure.MD [hunk 4]
```diff
@@ -133,35 +157,37 @@ You can deploy the template in three following ways:
...
 
-* <a href="https://github.com/Azure/azure-quickstart-templates/tree/master/jboss-eap-clustered-vmss-rhel" target="_blank"> JBoss EAP on RHEL (clustered, VMSS)</a> - This Azure template deploys a web application called eap-session-replication on JBoss EAP 7.2/EAP 7.3 cluster running on RHEL 7.7/8.0 VMSS instances.
+* <a href="https://azure.microsoft.com/en-us/resources/templates/jboss-eap-clustered-vmss-rhel/"> JBoss EAP on RHEL (clustered, Virtual Machine Scale Set)</a> - This Azure template deploys a web application called eap-session-replication on JBoss EAP 7.2 or 7.3 cluster running on RHEL 7.7 or 8.0 Virtual Machine Scale Set instances.
 
 ## Resource Links:
 
-* Explore [JBoss EAP on Azure Red Hat OpenShift](https://azure.microsoft.com/services/openshift/)
-* Explore [JBoss EAP on App Service Linux](https://docs.microsoft.com/azure/app-service/quickstart-java)
-* [Azure for Java developers documentation](https://github.com/JasonFreeberg/jboss-on-app-service)
+* [Azure Hybrid Benefits)(https://docs.microsoft.com/azure/virtual-machines/windows/hybrid-use-benefit-licensing)
```
Comment (Suraj2093) [inline ms-docs/JBoss EAP on Azure.MD:180]: Edited the hyperlink mapping for Azure Hybrid Benefits

File: ms-docs/JBoss EAP on Azure.MD (hunk 3 above)
Comment (jamesfalkner) [reply ms-docs/JBoss EAP on Azure.MD:41]: I think there's a typo (extra set of parentheses arounc `VS Code`) -- see the screenshot below showing other links render correctly, but the VS Code one does not:

image

Comment (jamesfalkner) [reply ms-docs/JBoss EAP on Azure.MD:41]: At the beginning of the paragraph, can you spell out "Red Hat Migration Toolkit for Applications" and make it a link to `https://developers.redhat.com/products/mta/overview`

Comment (Suraj2093) [reply ms-docs/JBoss EAP on Azure.MD:41]: I have made the changes you mentioned.

**Pull request closed without merging**